com.zsmartsystems.zigbee.dongle.ember/src/main/java/com/zsmartsystems/zigbee/dongle/ember/ZigBeeDongleEzsp.java

```diff
@@ -1040,8 +1052,6 @@ public void run() {
                 if (linkState) {
                     logger.debug("Ember: Link State up running");
 
-                    EmberNcp ncp = getEmberNcp();
-
```

Comment:
I'm uncomfortable with reusing the same instance for the reasons mentioned already. It might be ok if we can be sure that they always run in the same thread, but otherwise I think there should be a new instance.